tests/testHsm.py

```diff
@@ -306,6 +313,31 @@ def testHsmSourceMoments(self):
             self.assertAlmostEqual(xy, expected.getIxy(), SHAPE_DECIMALS)
             self.assertAlmostEqual(yy, expected.getIyy(), SHAPE_DECIMALS)
 
+    def testHsmRoundMoments(self):
+        for (i, imageid) in enumerate(file_indices):
+            source = self.runMeasurement("ext_shapeHSM_HsmRoundMoments", imageid, x_centroid[i], y_centroid[i],
+                                         sky_var[i])
+            x = source.get("ext_shapeHSM_HsmRoundMoments_x")
+            y = source.get("ext_shapeHSM_HsmRoundMoments_y")
+            xx = source.get("ext_shapeHSM_HsmRoundMoments_xx")
+            yy = source.get("ext_shapeHSM_HsmRoundMoments_yy")
+            xy = source.get("ext_shapeHSM_HsmRoundMoments_xy")
+            flux = source.get("ext_shapeHSM_HsmRoundMoments_Flux")
+
+            # Centroids from GalSim use the FITS lower-left corner of 1,1
```

Comment:
Shouldn't we correct for that in the algorithm, to present a unified set of coordinates in the catalog?

Reply:
No, the issue here is the difference in convention between LSST and galsim.  By default, the bounds of an image are constructed in galsim with (1,1) in the lower-left corner while LSST uses (0,0).  The precomputed galsim values we have in the test were measured using this default.  If we wanted to change this, we could change the bounds of the LSST image that is created in the test to match galsim or recompute the galsim values, but I don't think it is worth the effort.